lib/tracecontext/index.js

```diff
@@ -63,6 +64,22 @@ class TraceContext {
   toTraceParentString () {
     return this.traceparent.toString()
   }
+
+  propagateTraceContextHeaders (carrier, contextFrom, setter) {
```

Comment:
The methods [`toTraceParentString()`](https://github.com/elastic/node-traceparent/blob/4d97700fca219dfde202c0bb7a839f7792cb8b7a/index.js#L147-L149) and [`contextFrom.toTraceStateString()`](https://github.com/elastic/apm-agent-nodejs/blob/f7e34a62771d192f7bdb138224beb6fcba6743d2/lib/tracecontext/tracestate.js#L210-L227) appear to be rather expensive. Probably better to store them in a local variable than calling them several times.

Reply:
Feels a little like a premature optimization but it's easy enough to do.  Done in 05ed2ed6790b96705490a86a63461713d688fc6c